packages/tools/create-kadena-app/README.md

```diff
@@ -11,33 +12,45 @@ Create Kadena App is an easy way to get started with building apps using kadena.
 
 <hr>
 
-The create-kadena-app CLI tool enables you to quickly start a new application that has Kadena Blockchain integration set up and ready to go. The application supports Typescript and makes use of `@kadena/client` and `@kadena/pactjs-cli`.
+The create-kadena-app CLI tool enables you to quickly start a new application
+that has Kadena Blockchain integration set up and ready to go. The application
+supports Typescript and makes use of `@kadena/client` and `@kadena/pactjs-cli`.
 
-The application is backed by a [**smart contract**](##The-Pact-smart-contract) written in Pact that is included for convenience and also deployed on the Kadena Blockchain, so you have a working application from the start.
+The application is backed by a [**smart contract**][1] written in Pact that is
+included for convenience and also deployed on the Kadena Blockchain, so you have
+a working application from the start.
 
 The two most common blockchain use cases are covered in this starter app:
+
 - Reading a message from the chain (requires no transaction).
-- Writing a message on the chain, which includes signing and sending a transaction on chain and waiting for it to be mined.
+- Writing a message on the chain, which includes signing and sending a
+  transaction on chain and waiting for it to be mined.
 
-[Chainweaver](https://docs.kadena.io/basics/chainweaver/chainweaver-user-guide) is used for signing transactions, since it's the Official Kadena wallet for advanced blockchain usage and smart contract development.
+[Chainweaver][2] is used for signing transactions, since it's the Official
+Kadena wallet for advanced blockchain usage and smart contract development.
 
 ## Supported Templates
+
 <hr>
 Create Kadena App supports a number of well known and widely used frameworks to choose from when starting a new project. The following project templates are currently available:
 
-- [Nextjs](https://nextjs.org/)
-- [Vuejs](https://vuejs.org/) - *(Under development, coming soon)*
-- [Angular](https://angular.io/) - *(Under development, coming soon)*
+- [Nextjs][3]
+- [Vuejs][4] - _(Under development, coming soon)_
+- [Angular][5] - _(Under development, coming soon)_
```

Comment:
The issue with spaces for numbered lists (two spaces) and bullet lists, this feels inconsistent